app/src/main/java/io/apicurio/registry/ccompat/store/RegistryStorageFacadeImpl.java

```diff
@@ -71,13 +71,11 @@
     }
 
     @Override
-    public List<SubjectVersion> getSubjectVersions(int globalId) {
+    public List<SubjectVersion> getSubjectVersions(int contentId) {
```

Comment:
@carlesarnal @EricWittmann this facade is shared with the IBM api no? will it be an issue for ibm if we change this to use contentId

Reply:
Is it really?  I didn't realize that.  If that's true then yes that would be an issue for them.

Reply:
I can't see this facade being used in the IBM API. We're directly injecting the storage. @famartinrh do you have a reference to that?

Reply:
I've never been happier that @famartinrh was wrong.  :)  Whew!!